.gitignore

```diff
@@ -1,4 +1,11 @@
 /Manifest.toml
 /docs/build/
 /docs/src/source/
-.vscode/
\ No newline at end of file
+.vscode/
+
```

Comment:
Probably revert this file

Reply:
We aren't okay with having that in gitignore? I added it so that my .vscode files wouldn't get added by mistake.

Reply:
No its all the lines below, sorry doesnt show in the comment

Reply:
Oh yeah lol - that should go